src/program/content/trakt.py

```diff
@@ -73,18 +77,31 @@ def missing(self):
             logger.log("TRAKT", "Trending fetching is disabled.")
         if not self.settings.fetch_popular:
             logger.log("TRAKT", "Popular fetching is disabled.")
+        if not self.settings.fetch_most_watched:
+            logger.log("TRAKT", "Most watched fetching is disabled.")
```

Comment:
Honestly, we can ditch these 3 log messages, as they dont really provide any value to the user. I used to use them before as a way to show users whats available.. we can remove these 3 log messages though 👌